packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -82,18 +84,22 @@ export class AtomicRatingFacet
    */
   @Prop() public field!: string;
   /**
-   * The number of stars to request for this facet.
+   * The number of intervals to split the index for this facet.
    */
-  @Prop() public numberOfStars = 5;
+  @Prop() public numberOfIntervals = 5;
   /**
-   * The maximum value of the field. This value is used to normalize the field values with the number of stars.
+   * The maximum value of the field. This value is also used as the number of icons to be displayed.
    */
   @Prop() public maxValueInIndex = 5;
   /**
    * Whether to display the facet values as checkboxes (multiple selection) or links (single selection).
    * Possible values are 'checkbox' and 'link'.
    */
   @Prop() public displayValuesAs: 'checkbox' | 'link' = 'checkbox';
+  /**
+   * The icon used to display the rating.
+   */
+  @Prop() public icon = Star;
```

Comment:
Have you tried a different icon? What does it look like? 📷

Reply:
Fishes for Bass pro shop:
![Screen Shot 2021-07-22 at 1 25 40 PM](https://user-images.githubusercontent.com/83350303/126682356-7f63db51-2750-412d-8522-68f95dede1d5.png)